build_tools/rename_zip.js

```diff
@@ -23,4 +23,13 @@ const oldName = `build/${osdJson.id}-${osdJson.opensearchDashboardsVersion}.zip`
 const newName = `build/${pluginName}-${packageJson.version}.zip`;
 
 console.log('rename ' + oldName + ' to ' + newName);
-shell(`mv ${oldName} ${newName}`);
+const os = require('os')
+
+if(os.type() == 'Windows_NT'){
```

Comment:
Can we remove the platform check and only use renameSync?

Reply:
Then will it work on Linux?

Reply:
Yes

Reply:
Ok then

Reply:
I removed